src/main/scala/it/unibo/authsim/library/dsl/policy/builders/PolicyAutoBuilder.scala

```diff
@@ -27,40 +27,41 @@ object PolicyAutoBuilder:
       new PolicyAutoBuilder[String]:
         override def generate: String =
           val generatedString: ListBuffer[Char] = ListBuffer.empty
+          val maxDefault: Int => Int = (min: Int) => min + 10

```

Comment:
10 is a magic number (in this case, is more linked to the variable name rather than the spurious value itself).

Reply:
In my opinion, it doesn't make sense to create a variable for 10 here.

Reply:
Reading the line, it is hard to extrapolate the meaning of the variables: `maxDefault` and `min` do not say anything about what they are about. Thus, 10 becomes a magic number (and this is the reason of my first comment).

If possible, try to change these variable names to explain better their functions (then, 10 will no longer be a magic number).